ooipy/request/authentification.py

```diff
@@ -41,7 +41,7 @@ def get_authentification():
 
     filename = "ooi_auth.txt"
     if os.path.isfile(filename):
-        file = open(filename, "r")
+        file = open(filename)
```

Comment:
Could you revert this back to use the `"r"` argument? It's always good to lean into the side of being explicit, rather than implicit. Thanks!

Reply:
This was one of the pyupgrade fixes ([redundant `open` modes](https://github.com/asottile/pyupgrade#redundant-open-modes)). The tool doesn't support rule disables or line disables.

Reply:
Hmm. okay that's fine then. Thanks!